sources/osg/FrameBufferObject.js

```diff
@@ -216,7 +218,14 @@ FrameBufferObject.prototype = MACROUTILS.objectInherit( GLObject.prototype, MACR
         var gl = this._gl;
 
         // apply on unit 1 to init it
-        state.applyTextureAttribute( 1, texture );
+        // make sure we do bind it whatever state stack
+        // texture is cached
+        state.applyTextureAttribute( 1, texture, true );
```

Comment:
if it's not sync there is a bigger problem to solve. I would prefer we fix the bigger problem if there is.
Could you describe why you need to force a bind ?